src/clj_kondo/impl/analysis.clj

```diff
@@ -187,6 +189,7 @@
                 :impl-ns impl-ns
                 :filename filename
                 :defined-by defined-by
+                :defined-by->lint-as defined-by->lint-as
```

Comment:
This formatting looks odd, too much extra spaces?

Reply:
There is the same in other places, I suppose you used some find-replace?

Reply:
I'm not sure what you mean